pkgs/applications/science/biology/kent/kent-utils.patch

```diff
@@ -0,0 +1,27 @@
+--- ./src/utils/bigWigCat/makefile.orig	2023-03-27 14:03:01.585896516 +0000
```

Comment:
This patch is not used anywhere, and doesn't belong to this folder - should be deleted.

Reply:
As I've understand it, kent disable patching and this is needed for this older version. I haven't figured out another way of doing it.